### DISCUSSION TITLE
Is this correct for explicit self relations? I inversed the @relation name so it makes more sense when adding relation

### DISCUSSION THREAD

**Discussion by Darren120** [prisma/prisma#10093] [Q&A]
```prisma
model User {
  SubscribedTo SubscribeInfo[ ]      @relation("SubscribedTo")
  SubscribedBy SubscribeInfo[ ]      @relation("SubscribedBy")
}

model SubscribeInfo {
  SubscribedToId Int
  SubscribedById Int
  createdAt DateTime @default(now())
  SubscribedTo User @relation("SubscribedBy",fields: [SubscribedToId], references: [Id], onDelete: Cascade, onUpdate: NoAction)
  SubscribedBy User @relation("SubscribedTo", fields: [SubscribedById], references: [Id], onDelete: Cascade, onUpdate: NoAction)
  @@id([SubscribedToId, SubscribedById])
  @@index([SubscribedById])
}
```

**Comment by austincrim** [comment]
Hey @Darren120 👋, this looks correct to me for an explicit relation. Let me know if you run into any problems 👍 .

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
